cmd/kube-proxy/app/server.go

```diff
@@ -145,6 +145,7 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.ConfigFile, "config", o.ConfigFile, "The path to the configuration file.")
 	fs.StringVar(&o.WriteConfigTo, "write-config-to", o.WriteConfigTo, "If set, write the default configuration values to this file and exit.")
 	fs.StringVar(&o.config.ClientConnection.Kubeconfig, "kubeconfig", o.config.ClientConnection.Kubeconfig, "Path to kubeconfig files with authorization information (the master location is set by the master flag).")
+	fs.StringVar(&o.config.TenantPartitionKubeConfig, "tenant-server-kubeconfig", o.config.TenantPartitionKubeConfig, "Comma separated string representing kubeconfig files point to tenant api servers.")
```

Comment:
with the new TenantPartitionKubeconfig parameter, the  ClientCOnnection.Kubeconfig parameter can be removed.

Reply:
rp connection seems needed as events (from kube-proxy) would still go there. Besides, in scale-up env, kubeconfig is better than other form as it is sort of k8s convention.